model/src/test/kotlin/OrtResultTest.kt

```diff
@@ -304,7 +304,7 @@ class OrtResultTest : WordSpec({
                 .shouldContainExactly("rule id")
         }
 
-        "drop resolved rule violations if omitResolved is true" {
+        "filter out issues which are resolved or below warning severity" {
```

Comment:
Here, basically the same remark as above applies.

Reply:
Do you have a proposal?